modules/engines/engine-hdbti/src/main/java/com/sap/xsk/hdbti/dao/XSKCSVRecordDao.java

```diff
@@ -111,13 +111,13 @@ public void deleteAll(List<String> ids, String tableName) throws SQLException {
         }
 
         try (Connection connection = dataSource.getConnection()) {
-          PersistenceTableModel tableMetadata = dbMetadataUtil.getTableMetadata(tableName,
-              XSKCommonsDBUtils.getTableSchema(dataSource, tableName));
-          if(null == tableMetadata){
-            logger.debug("Table with name [" + tableName + "] was not found.");
-            return;
-          }
-          String pkColumnName = tableMetadata.getColumns().get(0).getName();
+            PersistenceTableModel tableMetadata = dbMetadataUtil.getTableMetadata(tableName,
+                    XSKCommonsDBUtils.getTableSchema(dataSource, tableName));
```

Comment:
this is wrong, it will return the first schema in which table with this name exist, however this might not be the right schema, especially in case of multi tenant application with schema isolation. Please create an issue for this, so that we can fix it.

Reply:
Done.